getting-started/your-first-feature/3_messaging_ui.md

```diff
@@ -0,0 +1,5 @@
+The UI for messaging will consist of a new Messages panel in addition to the Follow and Network panel. This panel will have two parts:
```

Comment:
I would also put these code names, like Messages, Follow etc. in backticks.

Reply:
Added backticks in all the suggested places.